content/en/docs/ops/configuration/telemetry/monitoring-multicluster-prometheus/index.md

```diff
@@ -0,0 +1,166 @@
+instances in a few simple ways. For production use cases, it is recommended to secure access to each Prometheus endpoint
+with HTTPS, as well as taking appropriate precautions such as using an internal load-balancer instead of a publicly-accessible
+endpoint and/or properly configuring firewall rules.
+{{< /warning >}}
+
+Istio provides a way to expose cluster services externally via [Gateways](/docs/reference/config/networking/gateway/).
+You can configure an ingress gateway for the cluster-local Prometheus, providing external connectivity to the in-cluster
+Prometheus endpoint.
+
+For each cluster, you will need to follow the appropriate instructions from the [Remotely Accessing Telemetry Addons](/docs/tasks/observability/gateways/#option-1-secure-access-https) task. And you
+**SHOULD** establish secure (HTTPS) access.
+
+After that, you will need to configure your external Prometheus to access the cluster-local Prometheus(es). This can be achieved with configuration like
+the following (replacing the gateway address and cluster name):
+
+{{< text yaml >}}
+scrape_configs:
+  - job_name: 'federate-{{CLUSTER_NAME}}'
+    scrape_interval: 15s
+
+    honor_labels: true
+    metrics_path: '/federate'
+
+    params:
+      'match[]':
+        - '{job="pilot"}'
+        - '{job="envoy-stats"}'
+
+    static_configs:
+      - targets:
+        - '{{GATEWAY_IP_ADDR}}:15030'
+        labels:
+          cluster: {{CLUSTER_NAME}}
+{{< /text >}}
+
+Notes:
+
+* `CLUSTER_NAME` should be set to the same value which is used to create the cluster (set via `values.global.multiCluster.clusterName`).
+
+* No Authentication to the Prometheus endpoint(s) is provided. This means that anyone will be able to query your
+cluster-local Prometheus(es). This may not be desirable.
+
+* Without proper HTTPS configuration of the gateway, everything is being transported via plaintext. This may not be
+desirable.
+
+### Production Prometheus from one of the clusters
+
+If you desire to run a Prometheus in one of the clusters, you will need to establish connectivity from the production
+instance of Prometheus to each of the cluster-local Prometheus(es) within the mesh.
+
+This is really just a customization of the process for external federation. You will need to establish the `Gateway`,
+`VirtualService`, and `DestinationRule` in each remote cluster as above in External Production Prometheus.
+
+{{< image width="80%"
+    link="./in-mesh-production-prometheus.svg"
+    alt="Architecture of in-mesh Production Prometheus for monitoring multicluster Istio."
+    caption="In-mesh Production Prometheus for monitoring multicluster Istio"
+    >}}
+
+You need to configure your production Prometheus to access both of the *local* and the *remote* Prometheus(es). This can be achieved
+with config like the following for the *remote* clusters (replacing the service name and cluster name for each cluster):
```

Comment:
```suggestion
Configure your production Prometheus to access both the *local* and the *remote* Prometheus instances. This can be achieved
by adding a configuration like the following for the *remote* clusters (replacing the service name and cluster name for each cluster):
```